frontend/src/scenes/insights/Insights.js

```diff
@@ -213,6 +213,7 @@ function _Insights() {
                         They each have generic logic that updates the url
                         */}
                                 <Card
+                                    className="ph-no-capture"
```

Comment:
I think there's a bunch of stuff here that doesn't need to be obscured (e.g. filters, error/timeout messages, filters tabs, ...)

Reply:
oh maybe this lingered - thought I had updated. filters I think should be.

Reply:
By filters I meant like weekly/daily/monthly, type of graph, ...., not actual filters, sorry for the confusion. I believe filters we do obscure very targeted.